code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -140,7 +182,7 @@
 	if(!..())
 		return FALSE
 	var/head_check = 0
-	for (var/mob/new_player/player in mode.living_players)
+	for(var/mob/player in mode.living_players)
```

Comment:
ree

Reply:
I always thought you were the one who was making those! I wonder who we have to blame for all of them. Someone keeps doing it with ifs too, like `if ()`